logtail.go

```diff
@@ -57,7 +57,7 @@ func (st SealTail) String() string {
 		st.LogActivity.UTC().Format(time.RFC3339),
 	)
 	if st.LastID != "" {
-		return fmt.Sprintf(
+		return s + fmt.Sprintf(
```

Comment:
thats a bit inefficient

why not add %s to the format bstring and 's' as the first argument?
`
fmt.Sprintf("%s, tag activity: %v",
			s, st.TagActivity.UTC().Format(time.RFC3339),
		)
)
`

Reply:
Updated (and all the other places that it occurs)